### PR TITLE
Desktop version 1

### PR DESCRIPTION
**Pull request by 0sugo**
In this pr I did the following :
Created a new desktop version of my personal portfolio that was originally in mobile version using a media query.
I made sure the site is responsive by adding a break-point at 768px.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -478,11 +485,209 @@ input {
...
+  }
+
+  .recent-works {
+    margin-top: 3%;
+  }
+
+  .recent-works-cards {
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: row;
```
Comment (NduatiKagiri) [inline styles.css]: - [x] Please note that you are required to lay out your work cards using `display: grid;`. The Flexbox box should only be used to lay out the cards` contents. This has been specified in the project requirements.

File: styles.css [hunk 2]
```diff
@@ -478,11 +485,209 @@ input {
...
+    flex-wrap: wrap;
+    justify-content: center;
+    margin: 0;
+    height: 988px;
+    width: auto;
+  }
+
+  .works {
+    width: 30.33%;
+  }
```
Comment (NduatiKagiri) [inline styles.css]: - [x] Kindly add a hover effect on your cards. Please note from the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468&t=h5L9p23oW5s1BP0W-0) that the cards' border should be `#d0d9d4` but changes to `
#60c095` on hover.

File: styles.css [hunk 3]
```diff
@@ -478,11 +485,209 @@ input {
...
+
+  .framework-card {
+    width: 33%;
+    height: auto;
+  }
+
+  .skills-card {
+    width: 33%;
+    height: auto;
+  }
```
Comment (NduatiKagiri) [inline styles.css]: - [x] Kindly add a hover effect to these cards. Please note from the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468&t=h5L9p23oW5s1BP0W-0) that the cards' border should be `#d0d9d4` but changes to `
#60c095` on hover.

File: styles.css [hunk 4]
```diff
@@ -478,11 +485,209 @@ input {
...
+    text-align: start;
+    align-items: center;
+    line-height: 42px;
+    margin-top: -30px;
+  }
+
+  .form-inputs {
+    display: flex;
+    flex-direction: column;
+  }
```
Comment (NduatiKagiri) [inline styles.css]: - [x] Kindly add the correct on-focus effect on your form inputs. Please note from the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468&t=h5L9p23oW5s1BP0W-0) that all the inputs' borders should be `#d0d9d4` but changes to `#60c095` on focus.

**Review by NduatiKagiri — CHANGES_REQUESTED** [review]
Hi @0sugo 👋 ,

# CHANGES REQUIRED ♻️  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Almost there

## Highlights ✔️ 
✅ No Linters errors
✅ GitHub Flow observed

## Required Changes ♻️

- [ ] Please align your buttons' text as in the [Figma Template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468&t=h5L9p23oW5s1BP0W-0). I have noted that all your buttons' texts align to the left while in the template they are centered. Kindly fix that on all your buttons.
  - Your buttons:
    ouga1
  
  - Figma design buttons:
    ouga2


- [ ] _Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by 0sugo** [comment]
@NduatiKagiri Buttons on my end seem to be aligned center kindly may I know at what resolution do they misbehave?

File: styles.css (hunk 3 above)
Comment (0sugo) [reply styles.css:646]: @NduatiKagiri I cant see this design in the figma. The cards don't have a border.

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @0sugo, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Fair README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 



## Optional suggestions

- [OPTIONAL] Some headings are missing in your README file such as `Key Features`. Kindly add it and 1-3 appropriate information under it.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**